cmd/res/configuration.toml

```diff
@@ -11,6 +11,20 @@ LogLevel = "INFO"
       password = ""
   [Writable.Reading]
   ReadingUnits = true
+  [Writable.Telemetry]
+    Interval = "30s"
+    PublishTopicPrefix  = "edgex/telemetry" # /<service-name>/<metric-name> will be added to this Publish Topic prefix
+      [Writable.Telemetry.Metrics] # All service's metric names must be present in this list.
+      # Device SDK provided Service Metrics
+      EventsSent = true
+      ReadingsSent = true
+      # Common Security Service Metrics
+      SecuritySecretsRequested = true
+      SecuritySecretsStored = true
+      SecurityConsulTokensRequested = true
+      SecurityConsulTokenDuration = true
```

Comment:
These need to be set to false. Yep, I need to fix that on Device MQTT. ;-)

Reply:
done